README.md

```diff
@@ -69,8 +68,8 @@ Software listed below should be available in [PATH](https://en.wikipedia.org/wik
    git clone git@github.com:paliarush/magento2-vagrant-for-developers.git vagrant-magento
    ```
  
- 1. Copy [etc/composer/auth.json.dist](etc/composer/auth.json.dist) to `etc/composer/auth.json` and specify your [GitHub OAuth token](https://github.com/settings/tokens) there. See [API rate limit and OAuth tokens](https://getcomposer.org/doc/articles/troubleshooting.md#api-rate-limit-and-oauth-tokens) for more information
- 
+ 1. Copy [etc/composer/auth.json.dist](etc/composer/auth.json.dist) to `etc/composer/auth.json`
```

Comment:
This step should become optional now, auth.json should be copied form auth.json.dist programmatically in the same way config.yaml.dist is copied.